### PR TITLE
Fixed typo when creating GroundPrimitives.

### PR DESCRIPTION
**Pull request by tfili**
`GroundPrimitives` changed its interface to accept more than one `GeometryInstance` at some point and the dynamic geometries that used them weren't updated.

Not exactly sure how to test this, as I can't verify a primitive has any `GeometryInstance`s once its created by the dynamic geometry updater.

CC #4160 @hpinkos

### CODE REVIEW

File: Apps/Sandcastle/gallery/CZML Polygon - Intervals, Availability.html [hunk 1]
```diff
@@ -138,8 +138,10 @@
     }
 }];
 
-var viewer = new Cesium.Viewer('cesiumContainer');
-viewer.dataSources.add(Cesium.CzmlDataSource.load(czml));
+Cesium.GroundPrimitive.initializeTerrainHeights().then(function() {
```
Comment (mramato) [inline Apps/Sandcastle/gallery/CZML Polygon - Intervals, Availability.html]: Why is this needed here?  The Entity visualization system should already be doing this automatically for the user, if it's not then we need to add code to make that happen; not force users to do it themselves.

**Comment by tfili** [comment]
@mramato Should be good to go.

**Comment by mramato** [comment]
There are some failing tests.
